src/shared/discover-image.c

```diff
@@ -282,6 +282,44 @@ static int extract_image_basename(
         return 0;
 }
 
+static int image_update_quota(Image *i, int fd) {
+        _cleanup_close_ int fd_close = -EBADF;
+        int r;
+
+        assert(i);
+
+        if (IMAGE_IS_VENDOR(i) || IMAGE_IS_HOST(i))
+                return -EROFS;
+
+        if (i->type != IMAGE_SUBVOLUME)
+                return -EOPNOTSUPP;
```

Comment:
(We usually return -ENOTTY for inappropriate fstype and such?)

Reply:
Hmm, in discover-image.c EOPNOTSUPP is used consistently, hence please ignore.

Reply:
Yeah. Let's fix that later in another PR if necessary.